src/main/java/ch/njol/skript/conditions/CondContains.java

```diff
@@ -115,8 +116,14 @@ public boolean check(final Event e) {
 					} else {
 						if (container instanceof Inventory) {
 							final Inventory invi = (Inventory) container;
-							return items.check(e, (Checker<Object>) type
-									-> type instanceof ItemType && ((ItemType) type).isContainedIn(invi));
+							return items.check(e, (Checker<Object>) type -> {
+								if (type instanceof ItemType && ((ItemType) type).isContainedIn(invi)) {
```

Comment:
I'd rather have the following for readability's sake but if it happens to cause issues, it's fine the way it is now.
```suggestion
if (type instanceof ItemType)
	return ((ItemType) type).isContainedIn(invi);
else if (type instanceof ItemStack)
	return invi.contains((ItemStack) type);
```

Reply:
For sure, Ill do some changes and test it, and if it works all fine Ill push

Reply:
Okay I made the changes, tested, everything seems to work great 👍